emukit/bayesian_optimization/acquisitions/probability_of_improvement.py

```diff
@@ -67,3 +67,56 @@ def evaluate_with_gradients(self, x: np.ndarray) -> Tuple:
     @property
     def has_gradients(self):
         return isinstance(self.model, IDifferentiable)
+    
+    
+    
+class ProbabilityOfFeasibility(Acquisition):
+
+    def __init__(self, model: Union[IModel, IDifferentiable], jitter: np.float64 = np.float64(0)) -> None:
```

Comment:
Unless there is a specific reason to use `np.float64`, let's just use python's `float`

Reply:
The ProbabilityOfImprovement declares np.float64, too; this may be a good opportunity to change that as well to float.